src/components/common/Header/index.tsx

```diff
@@ -27,7 +33,11 @@ const Header = ({ onMenuToggle }: HeaderProps): ReactElement => {
       </div>
 
       <div className={css.logo}>
-        <SafeLogo alt="Safe Logo" height={29} className={css.logo} />
+        <Track {...OVERVIEW_EVENTS.HOME}>
```

Comment:
It links to welcome tho. Should we change the link to `/`?

Reply:
I will, it then matches `safe-react`.